frontend/src/components/Shared/CrossingMapPage/CrossingMapSearchBar.js

```diff
@@ -1,50 +1,262 @@
 import React, { Component } from 'react';
 import 'components/Shared/CrossingMapPage/CrossingMapSearchBar.css';
 import FontAwesome from 'react-fontawesome';
+import Autosuggest from 'react-autosuggest';
+import MapboxClient from 'mapbox';
+import CrossingMapSearchCrossingSuggestions from 'components/Shared/CrossingMapPage/CrossingMapSearchCrossingSuggestions';
+
+const mapboxClient = new MapboxClient(
+  'pk.eyJ1IjoiY3Jvd2VhdHgiLCJhIjoiY2o1NDFvYmxkMHhkcDMycDF2a3pseDFpZiJ9.UcnizcFDleMpv5Vbv8Rngw',
+);
+
+let autosuggestInput;
```

Comment:
Should `autosuggestInput` be part of `CrossingMapSearchBar`
?

Right now we're only going to show one search bar at a time, but just in case we do have multiple instances of `CrossingMapSearchBar`, I think it would be helpful to have the `autosuggestInput` ref live on the instance.